app/build.gradle

```
@@ -33,5 +33,8 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
-
+    // https://mvnrepository.com/artifact/io.reactivex.rxjava3/rxjava
+    implementation group: 'io.reactivex.rxjava3', name: 'rxjava', version: '3.0.7'
+    // https://mvnrepository.com/artifact/io.reactivex.rxjava3/rxandroid
+    implementation group: 'io.reactivex.rxjava3', name: 'rxandroid', version: '3.0.0'
```

Сделай над общим коммитом и чтобы выглядело однообразно
```suggestion

    // RxJava 3
    implementation 'io.reactivex.rxjava3:rxjava:3.0.7'
    implementation 'io.reactivex.rxjava3:rxandroid:3.0.0'
```